MapboxNavigation/RouteMapViewController.swift

```diff
@@ -297,13 +297,16 @@ class RouteMapViewController: UIViewController {
         let isLongRoad = routeProgress.distanceRemaining >= NavigationMapView.longManeuverDistance
         
         
+        //If the user is at the last turn maneuver, the map should zoom in to the default altitude.
+        let lastTurn = routeProgress.currentLeg.lastTurn
+        guard currentStep != lastTurn else { return setCamera(altitude: defaultAltitude) }
```

Comment:
What if the last step is 10 miles long? Should this happen when the user starts the last step or is near the end?

Reply:
Yeah, after talking with @ericrwolfe it seems I misunderstood what we meant by "final turn maneuver announcement". Will rework.